src/darker/help.py

```diff
@@ -89,4 +89,6 @@
 
 LINE_LENGTH = "How many characters per line to allow [default: 88]"
 
+TARGET_VERSION = "Python versions that should be supported. [default: py37]"
```

Comment:
It would be good to mention this only affects Black and not isort, Mypy, Pylint, Flake8 (or other linters), nor Flynt (support for Flynt is upcoming, see #308). In `black --help` I see the following, maybe use the same wording, so just add "by Black's output"?
```
  -t, --target-version [py33|py34|py35|py36|py37|py38|py39|py310|py311]
                                  Python versions that should be supported by
                                  Black's output. [default: per-file auto-
                                  detection]
```